tequilapi/endpoints/location.go

```diff
@@ -104,8 +103,7 @@ func (le *LocationEndpoint) GetLocation(writer http.ResponseWriter, request *htt
 //     schema:
 //       "$ref": "#/definitions/ErrorMessageDTO"
 func (le *LocationEndpoint) GetLocationByIP(writer http.ResponseWriter, request *http.Request, params httprouter.Params) {
```

Comment:
Looks like we don't need `GetLocationByIP` if we are removing requests for other's IP.
It's enough to have only `GetLocation`

Reply:
I blame rebase